packer/README.md

```diff
@@ -8,12 +8,17 @@ homebrew:
 
 or retrieving it from the download available at https://www.packer.io/downloads.html
 
-## To Run
+## To build locally
 
-Set up your ATLAS_USER and ATLAS_TOKEN environment variables with your username
-and API access token from Atlas, then invoke:
+    packer build -only=virtualbox-iso template.json
+    packer push -name=$ATLAS_USERNAME/rackhd template.json
```

Comment:
nope, intended to be there. You appear to always need to run one locally and manually (even with ATLAS integrations for results) before you can run in automatic mode, so left it in to be specific about ordering.